tests/Browser/BookRoomsTest.php

```diff
@@ -143,6 +144,48 @@ public function testCannotBookUnavailableTime()
         });
     }
 
+     /**
+     * Create a blackout, and then attempts to book a room during that time
+     * assert that the page doesn't change to the booking creation. 
+     */
+    public function testCannotBookDuringBlackout()
+    {
+        $this->browse(function (Browser $browser){
+
+            $admin = User::first();
+            $blackout = Blackout::factory()->create();
```

Comment:
the blackout is not being utilized?

Reply:
I realize I forgot to apply the blackout to the room

Reply:
documented in #641